### PR TITLE
22Mars: Add map and tiles

### PR DESCRIPTION
**Pull request by vandamm**
## Before clicking "Create"

- [x] Branch is derived from the latest `master`
- [x] Add the `pins` or `archive_alpha_games` label if this change will break existing games
- [x] Code passes linter with `docker compose exec rack rubocop -a`
- [x] Tests pass cleanly with `docker compose exec rack rake`

### Explanation of Change
First part of 22Mars implementation: only the map.
Does not include random map generation now.

### CODE REVIEW

**Comment by roseundy** [comment]
You should be running rubocop before submitting a PR.

**Comment by vandamm** [comment]
Sorry, on my machine it never finished but showed only passing tests. Will remove local env completely and try again

**Comment by vandamm** [comment]
Now it's green

File: lib/engine/game/g_22_mars/game.rb [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+          %w[70 80p 90 100 110 120 130 145 160 175 195 215],
+          %w[60 70p 80 90 100 110 120 130 145 160 175],
+          %w[50 60p 70 80 90 100 110 120 130 145],
+          %w[45 50p 55 65 75 85 95 105],
+          %w[40 45 50 55 65 75],
+          %w[35r 40r 45r 50r],
+          %w[0c],
+        ].freeze
+
+        CERT_LIMIT = { 2 => 12, 3 => 12, 4 => 9, 5 => 7 }.freeze
```
Comment (ollybh) [inline lib/engine/game/g_22_mars/game.rb:33]:
```suggestion
        CERT_LIMIT = { 3 => 12, 4 => 9, 5 => 7 }.freeze
```

File: lib/engine/game/g_22_mars/game.rb [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+            name: '2',
+            distance: 2,
+            price: 100,
+            rusts_on: '4',
+            num: 6,
+          },
+          {
+            name: '3',
+            distance: 3,
+            price: 17,
```
Comment (ollybh) [inline lib/engine/game/g_22_mars/game.rb:96]:
```suggestion
            price: 170,
```

File: lib/engine/game/g_22_mars/game.rb [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+            distance: 5,
+            price: 340,
+            rusts_on: '6+SS',
+            num: 3,
+          },
+          {
+            name: '6+S',
+            distance: 6,
+            price: 440,
+            rusts_on: '4',
```
Comment (ollybh) [inline lib/engine/game/g_22_mars/game.rb:118]:
```suggestion
```

File: lib/engine/game/g_22_mars/game.rb [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+            num: 4,
+          },
+          {
+            name: '6+SS',
+            distance: 6,
+            price: 560,
+            rusts_on: '4',
```
Comment (ollybh) [inline lib/engine/game/g_22_mars/game.rb:125]:
```suggestion
```

File: lib/engine/game/g_22_mars/game.rb [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+            num: 6,
+            discount: { '5' => 400 },
```
Comment (ollybh) [inline lib/engine/game/g_22_mars/game.rb:127]: The discount is the amount the price is reduced by, not the price that will be paid.
```suggestion
            discount: { '5' => 160 },
```

**Comment by vandamm** [comment]
@ollybh thanks, all fixed!

**Review by ollybh — APPROVED** [review]

**Pull request merged**